server/Controllers/changePassword.js

```diff
@@ -0,0 +1,45 @@
+import bcrypt from "bcrypt";
+import UserModel from "../Schemas/User";
+
+const validatePassword = (password) => {
+  const reg = /(?=^.{8,}$)((?!.*\s)(?=.*[A-Z])(?=.*[a-z]))((?=(.*\d){1,})|(?=(.*\W){1,}))^.*$/;
+  return reg.test(password);
+};
+
+const changePassword = async (req, res) => {
```

Comment:
Put all of that in `editUserInfo` (or `editUser` now), no need for a root just for that.
if a new password has been sent -> do all of that